WalletWasabi.Backend/wwwroot/css/style.css

```diff
@@ -934,133 +1114,167 @@ section:hover .pull-img-to-right, section:hover .pull-img-to-left {
     font-weight: bold;
     color: #fff;
 }
+
 .download-icon {
     height: 40px;
 }
-.download-icon.small {
-    height: 20px;
-    margin-left: 15px;
-    margin-top: 10px;
-    opacity: 0.5;
-}
+
+    .download-icon.small {
+        height: 20px;
+        margin-left: 15px;
+        margin-top: 10px;
+        opacity: 0.5;
+    }
+
 hr {
     background-color: var(--green);
     opacity: 1;
 }
+
 .m-100 {
     margin: 100px 0;
 }
+
 .card label {
     width: calc(100% - 30px);
 }
+
 @media (min-width: 576px) {
     body {
         --breakpoint: 540px;
     }
 }
+
 @media (min-width: 768px) {
     body {
         --breakpoint: 720px;
     }
 }
+
 @media (min-width: 992px) {
     body {
         --breakpoint: 960px;
     }
 }
+
 @media (min-width: 1200px) {
     body {
         --breakpoint: 1140px;
     }
 }
+
 @media (min-width: 1400px) {
     body {
         --breakpoint: 1320px;
     }
 }
+
 @media only screen and (max-width: 1199px) {
     .wallet-cubes {
         display: none;
     }
+
     .navbar-expand-lg .navbar-nav .nav-item {
         padding-left: 0.5em;
         padding-right: 0.5em;
     }
+
     .epic-text {
         font-size: 40px;
     }
 }
+
 @media only screen and (max-width: 991px) {
     h1 {
         font-size: 40px;
     }
+
     h2 {
         font-size: 24px;
     }
+
     .logo img {
         width: 100px;
     }
+
     .video-homepage video, .video-homepage .mobile_video_bg {
         height: 100vh;
         object-fit: cover;
     }
+
     .home-text-area {
         left: calc((100% - var(--breakpoint)) / 2);
     }
+
     .download-btn-area {
         position: absolute;
         left: calc((100% - var(--breakpoint)) / 2);
         transform: translateY(-50%);
     }
+
     .service-line {
         font-size: 20px;
         display: block;
     }
+
     .service {
         min-height: 200px;
         width: 100% !important;
     }
-    .service a {
-        padding: 100px 0;
-    }
+
+        .service a {
```

Comment:
Makes no sense to me.

Reply:
Only formatting but this is not a good way for that.
Since nothing changed on the website code just formatting, I recommend that you revert all the website changes and keep the original code formatting.

Reply:
> Makes no sense to me.

`.service a` is indented, because it is a sub-config of `.service {`, that is how I understand it.

> Since nothing changed on the website code just formatting, I recommend that you revert all the website changes and keep the original code formatting.

That is the purpose of this PR, just to format the code.

All the changes in the website code are just empty lines inserted or some indentation change.
I can still revert that if you guys don't like those, but IMO it makes sense to this formatting once and for all.

Reply:
On css this is not a right formatting, never seen like this before. Only the scss do something like this but there the items are inside the parent tag.

Reply:
@CAnorbo I reverted most of the website code changes except `index.html` & `onion-index.html`, the changes there make sense to me.

Reply:
Thank you!